### PR TITLE
[examples] update loss computation

### LINKED ISSUE #1273: [Dreambooth] Training script mean loss calculation question
Hello,

I was playing with Dreambooth training and noticed a thing [here](https://github.com/huggingface/diffusers/blob/0025626cd9feca260e05f295f272af4f7b9ce44a/examples/dreambooth/train_dreambooth.py#L606) 

Why did the author use this way of calculation mean of the loss? Is there any particular technical need to do so comparing to [this](https://github.com/huggingface/diffusers/blob/0025626cd9feca260e05f295f272af4f7b9ce44a/examples/dreambooth/train_dreambooth.py#L609)

### PR DESCRIPTION
**Pull request by patil-suraj**
Fixes #1273

### CODE REVIEW

**Review by pcuenca — APPROVED** [review]
Much easier to read :)

**Comment by patrickvonplaten** [comment]
Merging now to ping the slow tests.
